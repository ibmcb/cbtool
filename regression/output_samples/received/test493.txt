Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.254,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.0.254,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,predictablevm,auto,empty,default,none,check_boot_started=subscribe_on_starting --operation=vm-attach --cn=TESTCLOUD --uuid=8E27C283-D407-5DAF-940B-30B9FE5E1F31 --daemon". The process id is 15180.

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
Message "11111111-1111-1111-1111-111111111111 has started" published on channel "starting" (object "VM" ).
Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |150.236.130.83  |simhosta3                  |SUT                 |none    |none       |88688644-A3F7-5788-83E8-BC0D4176A31E
|vm_3          |tinyvm              |pico32      |154.11.42.39    |simhostb1                  |SUT                 |none    |none       |8EE69A5D-DBFE-5CA3-844C-650894B02A10
|vm_4          |db2                 |copper32    |203.87.210.63   |simhosta1                  |SUT                 |none    |none       |B2B72794-2049-517B-A739-D8E9DDA7C2FA
|vm_5          |netclient           |platinum64  |165.202.156.11  |simhostc4                  |SUT                 |none    |none       |75F12125-BDE4-54FF-8B18-7087FD5D32A0
|vm_6          |tinyvm              |pico32      |134.214.238.90  |simhosta1                  |SUT                 |none    |none       |D88BCB3D-645D-5F39-951D-05871E5D4397
|vm_7          |tinyvm              |pico32      |203.188.127.128 |simhostc3                  |SUT                 |none    |none       |08E6995E-7F7E-5819-AD2B-DC21FA7202F4
|vm_8          |tinyvm              |pico32      |239.168.250.94  |simhostb2                  |SUT                 |none    |none       |D02FCC7E-4E13-5E0F-9FF0-3E705302D723
|vm_9          |specwebback         |platinum64  |240.38.192.78   |simhostb0                  |SUT                 |none    |none       |B85F057F-A52C-5FE7-A32D-7B59FF245D18
|vm_10         |netclient           |iron32      |78.102.77.83    |simhosta2                  |SUT                 |none    |none       |98D7F4D5-D9CB-59DC-8DCA-60A3528B6649
|vm_11         |fen_hpc             |platinum64  |43.65.242.19    |simhostd4                  |LG                  |none    |none       |54D77C60-B66D-5246-9892-45C1DF944079
|vm_12         |tinyvm              |iron32      |148.108.192.191 |simhosta2                  |SUT                 |none    |none       |99E1DA3A-20A8-5FF9-A10E-9166CF832571
|vm_13         |tinyvm              |iron32      |135.130.244.208 |simhosta1                  |SUT                 |none    |none       |2935FBD2-C73F-5A47-B3A1-81B177DC9ADD
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhostc1                  |SUT                 |none    |none       |8E27C283-D407-5DAF-940B-30B9FE5E1F31

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |39.21M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13565837                      
|Total Connections Received                           |6495028                       
|Total Commands Processed                             |93979411                      
|Number of Keys                                       |396                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16113934320.0                 
|Data Size                                            |13823358128.0                 
|Index Size                                           |550383792                     
|Average Object Size                                  |6601.19199836                 
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |13                            
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |13                            
|VMs ARRIVED                                          |14                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |103                           

 ---------------- VMC-WIDE COUNTERS ----------------
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|1                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|6                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|3                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|3                             

