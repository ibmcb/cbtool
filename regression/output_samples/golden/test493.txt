Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.254,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.0.254,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,predictablevm,auto,empty,default,none,check_boot_started=subscribe_on_starting --operation=vm-attach --cn=TESTCLOUD --uuid=B49A87F9-0393-5A9C-B7F5-DFC40559513E --daemon". The process id is 11938.

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
Message "11111111-1111-1111-1111-111111111111 has started" published on channel "starting" (object "VM" ).
Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |175.58.241.182  |simhostb0                  |SUT                 |none    |none       |ADA59FDC-537B-5D15-92EE-F47450275C5F
|vm_3          |tinyvm              |pico32      |183.61.231.199  |simhostb1                  |SUT                 |none    |none       |43CB6C82-35D0-59D7-93D8-7CC9976794F3
|vm_4          |db2                 |copper32    |132.242.162.234 |simhostc3                  |SUT                 |none    |none       |91CBF7AB-2395-5B02-9D0F-A0779233415D
|vm_5          |netclient           |platinum64  |250.101.115.40  |simhostc3                  |SUT                 |none    |none       |028070F6-8B5D-5777-9381-761BD9149FB3
|vm_6          |tinyvm              |pico32      |216.188.184.69  |simhostb0                  |SUT                 |none    |none       |1E59F851-EA2A-583A-9BA0-17503BD3CADB
|vm_7          |tinyvm              |pico32      |236.85.97.65    |simhosta2                  |SUT                 |none    |none       |54E552D6-E456-5643-8D76-7E78AB14BF57
|vm_8          |tinyvm              |pico32      |96.88.163.75    |simhostc0                  |SUT                 |none    |none       |50C66AE0-80B5-58A0-A59E-755517DDC54A
|vm_9          |specwebback         |platinum64  |243.191.175.189 |simhostc0                  |SUT                 |none    |none       |FF7885D0-9DAB-5BB1-8E98-F7239A703A64
|vm_10         |netclient           |iron32      |162.34.47.147   |simhosta2                  |SUT                 |none    |none       |3F704FF7-F9EF-5CAE-8B46-4FE897EB8D64
|vm_11         |fen_hpc             |platinum64  |123.198.209.121 |simhostd2                  |LG                  |none    |none       |B86E9084-4132-531E-A25C-D221BD8D7E55
|vm_12         |tinyvm              |iron32      |153.211.241.249 |simhostb1                  |SUT                 |none    |none       |89CFAAAF-0824-58A6-918A-F36529D1A6EC
|vm_13         |tinyvm              |iron32      |59.67.210.203   |simhostb2                  |SUT                 |none    |none       |4FEC9B0A-6C01-58CE-8BB7-784174107FFD
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhosta1                  |SUT                 |none    |none       |B49A87F9-0393-5A9C-B7F5-DFC40559513E

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |39.20M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11741038                      
|Total Connections Received                           |6492845                       
|Total Commands Processed                             |92913340                      
|Number of Keys                                       |394                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13967671296.0                 
|Data Size                                            |12134935980.0                 
|Index Size                                           |502971168                     
|Average Object Size                                  |6344.74940068                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |13                            
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |13                            
|VMs ARRIVED                                          |14                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |103                           

 ---------------- VMC-WIDE COUNTERS ----------------
|35310FEB-1E59-53F1-A37C-0E4387DFF893 (simzone_a) VM RESERVATIONS|3                             
|DAEF899C-8C95-5B76-BE3F-58349C261E66 (simzone_d) VM RESERVATIONS|1                             
|8E18B916-CC95-566E-887E-3032BA7F835A (simzone_c) VM RESERVATIONS|4                             
|3076C295-3106-5ADC-8C71-D21083902D7A (simzone_b) VM RESERVATIONS|5                             

